src/main/java/com/aws/greengrass/componentmanager/ComponentStore.java

```diff
@@ -418,12 +422,22 @@ public long getContentSize() throws PackageLoadingException {
      * @throws PackageLoadingException if I/O error occurred
      */
     public long getUsableSpace() throws PackageLoadingException {
+#if ANDROID
+        try {
+            StatFs stat = new StatFs(nucleusPaths.componentStorePath().toString());
+            return stat.getAvailableBytes();
+        } catch (Exception e) {
+            throw new PackageLoadingException(
+                    "Failed to get usable disk space for directory: " + nucleusPaths.componentStorePath(), e);
+        }
+#else
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, delete these tokens

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.